client/rpc_sender.go

```diff
@@ -83,6 +84,13 @@ func newRPCSender(server string, context *base.Context, retryOpts retry.Options,
 // and been executed successfully. We retry here to eventually get through with
 // the same client command ID and be given the cached response.
 func (s *rpcSender) Send(ctx context.Context, ba roachpb.BatchRequest) (*roachpb.BatchResponse, *roachpb.Error) {
+	timeout := 5 * s.retryOpts.MaxBackoff
```

Comment:
Example: https://circleci.com/gh/cockroachdb/cockroach/10045; `TestMultiRangeEmptyAfterTruncate` fails (with a `Fatal`, which causes the stopper to not stop, hence all the leaked rpc clients).